cfg/gc/run-excalibur.sh

```diff
@@ -29,18 +29,18 @@ cd -
 
 # Run analysis with config from worker node tmp dir
 echo "#### Starting Artus..."
-_TME_FILE='artus.tme'
+_TME_FILE='excalibur.tme'
 if [[ $(which time) ]]; then
 	# GNU time allows for extended format - use IEEE POSIX.2 as base
-	$(which time) -o ${_TME_FILE} -f 'real %e\nuser %U\nsys %S\nrss %M\nmajor_pfault %F\nmajor_pfault%R\nwaits %w' @WORKPATH@/artus @ARTUS_CONFIG@ || exit 1
+	$(which time) -o ${_TME_FILE} -f 'real %e\nuser %U\nsys %S\nrss %M\nmajor_pfault %F\nmajor_pfault%R\nwaits %w' excalibur @ARTUS_CONFIG@ || exit 1
```

Comment:
The variable `@WORKPATH@` was used to strictly define the `artus` executable to run. Calling plain `excalibur` will look it up in `$PATH` (etc.).

As a minor issue, this requires that `excalibur` is actually in the path. As a major issue, this means that *all* excalibur runs use the same executable, possibly switching when recompiling during a run. Most importantly, this makes it impossible to just rerun an old workdir to recreate or append output.